Makefile

```diff
@@ -69,12 +70,12 @@ $(CCTZ_LIB): $(CCTZ_OBJS)
 install: install_hdrs install_lib
 
 install_hdrs: $(CCTZ_HDRS)
-	$(SUDO) mkdir -p $(PREFIX)/include
-	$(SUDO) cp -p $? $(PREFIX)/include
+	$(SUDO) mkdir -p $(DESTDIR)/$(PREFIX)/include
```

Comment:
`DESTDIR` is a common override that is used in `autotools` style makefiles. See the [documentation](https://www.gnu.org/software/automake/manual/html_node/DESTDIR.html).

The reason to have a seperate `PREFIX` and `DESTDIR` is that the prefix is the installation prefix but the DESTDIR allows installation into a different location. This is especially useful when cross-compiling as the `PREFIX` is actually `/usr` but the `DESTDIR` is something like `/home/user/cc-staging`. Then the `DESTDIR` is packaged up and expanded onto the target device.

Some projects use the `prefix` variable to understand where the project is to be installed eventually. This project doesn't use that at all in the C++ code but it would be nice if we can play nice with common `Makefile` conventions. Happy to remove it if you don't want it though :smile: